api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -106,7 +106,8 @@ message SubstitutionFormatString {
   //
   //   content_type: "text/html; charset=UTF-8"
   //
-  string content_type = 4;
+  string content_type = 4
+      [(validate.rules).string = {well_known_regex: HTTP_HEADER_VALUE strict: false}];
```

Comment:
This is a breaking change to the API, but IMHO it makes sense.
@htuch WDYT?

Reply:
Yeah I think this is fine.

Reply:
The rubric we use is "could there have been some reasonable use of this previously?". Here we're talking about something that is dictated by RFC, so that basically documented what it should do, which is what this constraint is adding, so it's fine.